cdk-infra/lib/bulkExport.ts

```diff
@@ -0,0 +1,315 @@
+                            resources: ['*'],
+                        }),
+                        new PolicyStatement({
+                            effect: Effect.ALLOW,
+                            actions: ['dynamodb:Query', 'dynamodb:GetItem'],
+                            resources: [exportRequestDynamoDBTable.tableArn],
+                        }),
+                        new PolicyStatement({
+                            effect: Effect.ALLOW,
+                            actions: ['kms:Decrypt'],
+                            resources: [dynamoDbKMSKey.keyArn],
+                        }),
+                    ],
+                }),
+            },
+        });
+
+        this.glueScriptsBucket = new Bucket(scope, 'glueScriptsBucket', {
+            encryption: BucketEncryption.S3_MANAGED,
+            serverAccessLogsBucket: fhirLogsBucket,
+            serverAccessLogsPrefix: 'GlueScriptsBucket',
+            blockPublicAccess: {
+                blockPublicAcls: true,
+                blockPublicPolicy: true,
+                ignorePublicAcls: true,
+                restrictPublicBuckets: true,
+            },
+        });
+
+        this.glueScriptsBucketHttpsOnlyPolicy = new BucketPolicy(scope, 'glueScriptsBucketHttpsOnlyPolicy', {
+            bucket: this.glueScriptsBucket,
+        });
+        this.glueScriptsBucketHttpsOnlyPolicy.document.addStatements(
+            new PolicyStatement({
+                sid: 'AllowSSLRequestsOnly',
+                effect: Effect.DENY,
+                principals: [new StarPrincipal()],
+                actions: ['s3:*'],
+                resources: [this.glueScriptsBucket.bucketArn, `${this.glueScriptsBucket.bucketArn}/*`],
+                conditions: {
+                    Bool: {
+                        'aws:SecureTransport': 'false',
+                    },
+                },
+            }),
+        );
+
+        this.bulkExportResultsBucket = new Bucket(scope, 'bulkExportResultsBucket', {
+            encryption: BucketEncryption.S3_MANAGED,
+            serverAccessLogsBucket: fhirLogsBucket,
+            lifecycleRules: [
+                {
+                    id: 'ExpirationRule',
+                    enabled: true,
+                    expiration: Duration.days(3),
+                },
+            ],
+            serverAccessLogsPrefix: 'BulkExportResultsBucket',
+            blockPublicAccess: {
+                blockPublicAcls: true,
+                blockPublicPolicy: true,
+                ignorePublicAcls: true,
+                restrictPublicBuckets: true,
+            },
+        });
+
+        this.bulkExportResultsBucketHttpsOnlyPolicy = new BucketPolicy(
+            scope,
+            'bulkExportResultsBucketHttpsOnlyPolicy',
+            {
+                bucket: this.bulkExportResultsBucket,
+            },
+        );
+        this.bulkExportResultsBucketHttpsOnlyPolicy.document.addStatements(
+            new PolicyStatement({
```

Comment:
Nit pick - AllowSSLRequestsOnly and blockPublicAccess can be reused.

Reply:
I've refactored to use this suggestion!